src/api/schema/gen.rs

```diff
@@ -0,0 +1,111 @@
+use std::fs;
+use vector::api::schema::build_schema;
+
+static INTROSPECTION_QUERY: &str = r#"
+query IntrospectionQuery {
```

Comment:
This is the full introspection query that retrieves all parts of the GraphQL schema, including a few of the features we don't currently use (like directives) but may in future.